src/chrome/content/rules/epsxe.com.xml

```diff
@@ -0,0 +1,17 @@
+<!--
+	Refused:
+		gerrit.epsxe.com
+		www.gerrit.epsxe.com
+		webmail.epsxe.com
+-->
+<ruleset name="epsxe.com">
+	<target host="epsxe.com" />
+	<target host="www.epsxe.com" />
+	<target host="cpanel.epsxe.com" />
+	<target host="greenimp.epsxe.com" />
+	<target host="www.greenimp.epsxe.com" />
+	<target host="mail.epsxe.com" />
```

Comment:
https://travis-ci.org/EFForg/https-everywhere/jobs/586220331#L325
Move to comment.